features/admin.logs.v1/components/remote-logging-config-form.tsx

```diff
@@ -0,0 +1,461 @@
+     */
+    logType: LogType;
+    /**
+     * Remote logging configuration such as `remoteUrl`, `connectTimeoutMillis`, etc.
+     */
+    logConfigData: RemoteLogPublishingConfigurationInterface;
+    /**
+     * Remote log publishing configs loading.
+     */
+    isLoading: boolean,
+    /**
+     * Errpr fetching log config.
+     */
+    error: AxiosError,
+    /**
+     * Callback to mutate the remote logging request.
+     */
+    mutateRemoteLoggingRequest: () => void;
+}
+
+/**
+ * Component to hold the remote logging configurations.
+ *
+ * @param props - Props injected to the component.
+ * @returns Remote logging component.
+ */
+export const RemoteLoggingConfigForm = ({
+    logType,
+    logConfigData,
+    mutateRemoteLoggingRequest,
+    isLoading,
+    error,
+    "data-componentid": componentId = "remote-logging-config-form"
+}: RemoteLoggingConfigFormProps): ReactElement => {
+
+    const [ showDeleteConfirmationModal, setShowDeleteConfirmationModal ] = useState<boolean>(false);
+    const [ logConfig, setLogConfig ] = useState(null);
+
+    const dispatch: Dispatch = useDispatch();
+
+    const { t } = useTranslation();
+
+    const [ testNow, setTestNow ] = useState(false);
+
+    // Conditionally call the hook when shouldFetch is true
+    const { isLoading: isTesting, mutate: mutateTest } =
+        useTestRemoteLogPublishingConfiguration(testNow, logType);
+
+    useEffect(() => {
+        if (error) {
+            return;
+        }
+        if (logConfigData && (!logConfigData.publishInterval || logConfigData.publishInterval < 15)) {
+            logConfigData.publishInterval = 15;
+        }
+        if (logConfigData && !logConfigData.connectTimeoutMillis) {
+            logConfigData.connectTimeoutMillis = 30000;
+        }
+        setLogConfig(logConfigData);
+    }, [ logConfigData, error ]);
+
+    const handleRemoteLoggingConfigUpdate = (remoteLogPublishConfig: RemoteLogPublishingConfigurationInterface) => {
+
+        if (!remoteLogPublishConfig.connectTimeoutMillis) {
+            remoteLogPublishConfig.connectTimeoutMillis = 1000;
+        }
+        remoteLogPublishConfig.logType = logType;
+        updateRemoteLogPublishingConfiguration(remoteLogPublishConfig, logConfig === null)
+            .then((): void => {
+                mutateRemoteLoggingRequest();
+                dispatch(
+                    addAlert<AlertInterface>({
+                        description: t(
+                            "console:manage.features.serverConfigs.remoteLogPublishing.notification.success.description"
+                        ),
+                        level: AlertLevels.SUCCESS,
+                        message: t(
+                            "console:manage.features.serverConfigs.remoteLogPublishing." +
+                            "notification.success.message"
+                        )
+                    })
+                );
+            })
+            .catch((_err: AxiosError) => {
```

Comment:
From API we are throwing is an `IdentityAppsApiException`. Therefore use it here.
Please check other places as well.

```suggestion
            .catch((_err: IdentityAppsApiException) => {
```